### ISSUE TITLE
Missing section on the settings dropdown

### ISSUE THREAD

**Issue by danielo515** [elkarbackup/elkarbackup#633]
### Describe the bug

Hello, and thanks for this project, it is exactly what I was looking for. I just tried it and works like a charm, however there are a couple of small problems. I found workarounds for them, but it will be better if they get addressed.

One is related to the SSH key download. According to docs, there is a section on settings called "Manage parameters", but on my installation there is no such section as you can see on the screenshot: 
<img width="384" alt="image" src="https://user-images.githubusercontent.com/2270425/156987783-280b3bd9-dc09-426d-b47e-64d170b3a3ba.png">

I managed to download the ssh key by directly accessing the URL mentioned on the docs, but it will be awesome if this section becomes available again.

The other problem is that, despite accessing the app through ssh sometimes (for example, after login) it keeps redirecting to http. This can be problematic because a) it is less secure, and b) you may have elkarbackup behind a reverse proxy with only https configured

- Elkarbackup version: latest [sha256:1bc92c379399a662a5b529cafd1057ee2871a110377dd28da2efe5993608ed44]
- OS: [e.g Debian 10 Buster, Ubuntu 20.04 Focal]
- Deb package version / Docker image name: elkarbackup/elkarbackup:latest

### Expected vs actual behavior

I want to be able to download the SSH key from the web app and also not be redirected to http if I visit the app using https.

### Logs

NOT needed

**Comment by danielo515** [comment]
> About the ssh(?) connection, I have no clue. Are you referring to HTTPS connections? I do not believe that the docker container by itself supports HTTPS connections.

No, when I talk about ssh I'm referring to SSH connections. elkarbackup, in order to make a backup of a remote server it needs to connect through SSH to it, and for that to be possible you need to download elkarbackup's ssh key and upload it to the servers you want to target.
So not everything in "Manage PArameters" it's about things you can configure through env vars, so the section is still valuable even if it is not complete.



> The other problem is that, despite accessing the app through ssh sometimes (for example, after login) it keeps redirecting to http. This can be problematic because a) it is less secure, and b) you may have elkarbackup behind a reverse proxy with only https configured

This sentence has a mistake, it should say `https sometimes`. So in this sentence everything is about https. The problem is not that I want elkarbackup to support https, my reverse proxy is taking care of that, the problem is that it does a redirection to http, which is breaking the end to end encryption if you don't realize that you have been redirected.

Thank you for taking over this project! It looks fantastic

**Comment by danielo515** [comment]
> Are you mounting the volume elkarbackup uses for storing its ssh key like it's suggested in https://github.com/elkarbackup/elkarbackup/blob/master/docker/docker-compose.yml#L13?

That is a bit implicit. I wish it was stated in the docs. Something like "Note, if you are using docker containers, please use a volume to get the ssh keys" or something like that.



> So you having a reverse proxy for TLS makes things clearer. I had a look at the code and could not find an instance where elkarbackup is using http:// as the protocol. But it might very well be, that I am missing something.
> 
> Can you give me an example URL where you see this redirect to http?

Maybe it is something more "implicit". Let's say you send a generic redirect, and because the communication between the proxy and the container is http, then by default it assumes the same protocol and host. Maybe there is a way to signal elkar backup what protocol they should use for the redirects, or to make a redirect that includes some header that takes into account the origin or something like that.

One simple example is the login page

**Comment by ixs** [comment]
Okay. I believe I found the issue. This is not happening with elkarbackup links but with redirects generated by the Symfony framework.

https://github.com/elkarbackup/elkarbackup/pull/686 is a PR that would probably fix this by allowing you to define the trusted proxies and have Symfony take protocol information (http vs. https) from their headers when building URLs.

Unfortunately this change does not work with our current Symfony release, we need to update that one first.

In the meantime, you could try modifying the code as described in https://symfony.com/blog/fixing-the-trusted-proxies-configuration-for-symfony-3-3 in your own docker image and see if that helps.
